### ISSUE TITLE
Enable Pickles to deal with ignored scenario examples in VsTest Result Provider

### ISSUE THREAD

**Issue by Lisa-Lee** [picklesdoc/pickles#400]
Hi there, we see an error "Pickles did not complete successfully Object reference not set to an instance of an object." when try to generate Pickles report, does anyone face the same issue? Here are some details:
Framework: SpecFlow
Test Runner: VSTest
Commend I am using: 
`pickles.exe /feature-directory=MyFeatures /output-directory=MyPicklesResults /documentation-format=Dhtml /test-results-format=vstest /link-results-file="MyVSTestResult.trx"`

Our solution: comment out all the unimplemented Scenario Outline 

Can anyone give a better solution? Thanks!

**Comment by dirkrombauts** [comment]
Hi,
What do you mean by "comment out all the unimplemented Scenario Outline"?

Could you make a small project that demonstrates the problem? Thanks!

**Comment by Lisa-Lee** [comment]
Thank you @dirkrombauts 
We need to change an unimplemented Scenario Outline to a Scenario, like this:
`
Scenario: Add two numbers
	Given I have entered <a> into the calculator
	And I have entered <b> into the calculator
	When I press add
	Then the result should be <c> on the screen

	#Examples: 
	#| a | b | c |
	#| 1 | 2 | 3 |
`
Then the report can be generated and all the other implemented Scenarios can be reported correctly.

I will create a sample project, sorry a bit busy these days.

**Comment by dirkrombauts** [comment]
Just a quick check about your scenario outlines - they should look like this:

```gherkin
Scenario Outline: Add two numbers
	Given I have entered '<aa>' into the calculator
	And I have entered '<bb>' into the calculator
	When I press add
	Then the result should be '<cc>' on the screen
	Examples: 
	| aa | bb | cc |
	| 1  | 2  | 3  |
```

A scenario outline starts with `Scenario Outline:`, not just `Scenario:`.

**Comment by Lisa-Lee** [comment]
Hi @dirkrombauts please have a look at the sample project: https://github.com/Lisa-Lee/specflow-mstest

Pickles report is not generated if:
- there is `Scenario Outline` which is not implemented in the SpecFlow features
- there are tags in the SpecFlow features
- execute tests by using `TestCaseFilter:"TestCategory=tagname"` in vstest command line

However, Pickles report can be generated if there is 'Scenario' which is not implemented, 
that was the reason I changed 'Scenario Outline' to 'Scenario' and comment out the 'Examples'

**Comment by dirkrombauts** [comment]
Thanks! I will take a detailed look at it as soon as I can (later this week).

**Comment by dirkrombauts** [comment]
I was able to reproduce and the issue. I'm waiting on the CI build to finish before I merge this fix into the develop branch.

**Comment by dirkrombauts** [comment]
Released in version [2.11.1](https://github.com/picklesdoc/pickles/releases/tag/v2.11.1).

**Comment by Lisa-Lee** [comment]
Thank you very much! @dirkrombauts 
I can see the issue is fixed in version 2.11.1.

**Comment by dirkrombauts** [comment]
Thanks for letting me know!
